src/Microsoft.FeatureManagement.AppInsightsTelemetryPublisher/Microsoft.FeatureManagement.AppInsightsTelemetryPublisher.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
+    <AssemblyOriginatorKeyFile>..\..\build\Microsoft.FeatureManagement.snk</AssemblyOriginatorKeyFile>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Should this be 9.0?

Reply:
Since we're on `net6.0`, maybe we could use `10.0`

Reply:
This was the default when I created the project. Perhaps I should simply remove it so it uses latest compatible? This is what the AspNetCore package does https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj#L18

What do y'all think?

Reply:
If there's no compiler errors to remove it I would remove it.